sdk/src/feature_set.rs

```diff
@@ -724,6 +724,10 @@ pub mod update_hashes_per_tick6 {
     solana_sdk::declare_id!("FKu1qYwLQSiehz644H6Si65U5ZQ2cp9GxsyFUfYcuADv");
 }
 
+pub mod allow_commission_decrease_at_any_time {
+    solana_sdk::declare_id!("9wWpKE7vL7EmZcDTPm8n9pPVGJLynedAdAxh3BzZnAoF");
```

Comment:
@joncinque you want to hold the key for this?

Reply:
Yeah no problem, I ground out `decoMktMcnmiq6t3u7g5BfgcQu91nKZr6RvMYf9z1Jb`